### PR TITLE
Sign Up Page Layout

### PR DESCRIPTION
**Pull request by paridhikhaitan**
Created the Sign Up Layout Page and need to merge it with Master to add the menubar etc. Also added rules for breakpoints, margins.

### CODE REVIEW

File: src/components/layouts/signup-layout/style.less [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@import '../../../styles/vars.less';
+@import '../../../styles/mixins.less';
+@import '../../../styles/fonts.less';
+
+.SignupLayout {
+  /* Fix bug where ant row overflows container */
+  .ant-row-flex {
+    margin-right: 0 !important;
```
Comment (martinmags) [inline src/components/layouts/signup-layout/style.less]: Do we need to !important to override ant-row-flex?

File: src/components/login-card/index.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ const LoginCard = (props: any) => {
     }, []);
     return (
       <Card className="card-login">
-        <p className="login-header">Login</p>
+        <h2>Login</h2>
```
Comment (martinmags) [inline src/components/login-card/index.tsx]: Did we have a standard for h and p tags?

File: src/components/signup-card/style.less [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
-@import '../../styles/fonts.css';
+@import '../../styles/fonts.less';
 @import '../../styles/mixins.less';
 
 // Signup Card
 .card-signup{
-  width: @large-card-width;
-  height: @large-card-height;
+  width: 100%;
+  height: 100%;
```
Comment (martinmags) [inline src/components/signup-card/style.less]: We should probably remove some of these variables if we are just going to use percentage units.

File: src/components/login-card/index.tsx (hunk 2 above)
Comment (martinmags) [reply src/components/login-card/index.tsx]: Oh nvm I see it. Naisu.

File: src/components/signup-card/style.less (hunk 3 above)
Comment (paridhikhaitan) [reply src/components/signup-card/style.less]: Yes, we can just let them stay during testing and remove them later

File: src/components/login-card/index.tsx (hunk 2 above)
Comment (StoneT2000) [reply src/components/login-card/index.tsx]: header and paragraph styling should be handled by the d4sd components thing

File: src/components/layouts/signup-layout/style.less (hunk 1 above)
Comment (StoneT2000) [reply src/components/layouts/signup-layout/style.less]: it’s supposedly a bug at the moment when using gutter and the ant row flex. Didn’t find another solution at the moment

**Pull request merged**